src/js/frontback.js

```diff
@@ -31,3 +31,21 @@ else {
   	console.log('Frontback: need frontback.postUrl set with endpoint.');
   }
 }
+
+// Paste Screenshot functionality
+document.onpaste = function(event){
```

Comment:
This looks like it might hook on to paste for any page that Feedback appears on regardless of whether it's open/active or not. Is that true? Is it possible to attach it to the element that's the modal with the screenshot, or attach and detach the event listener when that modal appears and disappears?